en/services/camera_def.md

```diff
@@ -248,6 +248,16 @@ This indicates to the GCS that when the `CAM_MODE` parameter is set to _Video_,
 
 This example also tells the GCS not to display this parameter to the user (`control=“0”`). Camera Mode is a standard parameter defined in the [CAMERA_INFORMATION](../messages/common.md#CAMERA_INFORMATION) message and it’s handled by the GCS in that way. The parameter definition above was created in order to tell the GCS the rules that are applied when changes to the camera mode occur.
 
+#### Param Ranges
+
+It is also possible to define param ranges without individual options, but by specifiying the minimum, maximum, and optionally a step size:
+
+```
+<parameter name="CAM_APERTURE" type="float" default="2.8" min="2.8" max="14.0" step="0.1">
+    <description>ISO</description>
```

Comment:
1. Aperture isn't ISO
2. Can we show the "equivalent"? I.e. the other way of doing it?

I.e. would this be the same as (?):

```xml
<parameter name="CAM_APERTURE" type="float" default="2.8">
    <description>Aperture ..</description>
    <options>
      <roption name="2.8" value="2.8" />
      <roption name="2.9" value="2.9" />
...
      <roption name="14" value="14" />
    </options>
</parameter>
```

3. Can I use this format in `parameterrange` as well?
4. If I do it this way, can I define exclusions etc?

Reply:
1. Sorry, copy paste is wrong there.
2. We can show the equivalent, if you want.
3. Don't think so, at least it doesn't look implemented.
4. Probably not :thinking:

Reply:
@julianoes I've changed to capture all this, can you check that the update is OK?